src/Compilers/CSharp/Portable/Symbols/NullableAnnotation.cs

```diff
@@ -32,5 +32,10 @@ internal enum NullableAnnotation : byte
         /// Type is annotated with '?' - string?, T?.
         /// </summary>
         Annotated,
+
+        /// <summary>
+        /// Used for indexed type parameters and used locally in override/implementation checks.
```

Comment:
>Used for indexed type parameters and used locally in override/implementation checks. [](start = 12, length = 84)

Should probably provide details about how exactly it ignored. So that one could predict the impact of usage of this annotation